### PR TITLE
Minor cleanups to process handling

### PR DESCRIPTION
**Pull request by jduck**
* Fixed 'dpk' handling (signal wasn't being groked)
 * Mostly comments added
 * Some eprintfs added

### CODE REVIEW

File: libr/core/cmd_debug.c [hunk 1]
```diff
@@ -421,14 +421,16 @@ static void cmd_debug_pid(RCore *core, const char *input) {
 		/* XXX: not for threads? signal is for a whole process!! */
 		/* XXX: but we want fine-grained access to process resources */
 		pid = atoi (input+2);
-		ptr = strchr (input, ' ');
-		sig = ptr? atoi (ptr+1): 0;
 		if (pid > 0) {
+			ptr = input+2;
+			while (*ptr==' ') ptr++;
```
Comment (radare) [inline libr/core/cmd_debug.c]: use r_str_chop_ro or something like that. (ptr = r_str_chop_ro(input + 2))

File: libr/core/cmd_debug.c [hunk 2]
```diff
@@ -421,14 +421,16 @@ static void cmd_debug_pid(RCore *core, const char *input) {
...
+			ptr = strchr (ptr, ' ');
+			sig = ptr? atoi (ptr+1): 0;
```
Comment (radare) [inline libr/core/cmd_debug.c]: missing spaces around +

**Comment by radare** [comment]
looks good to merge after you fix the comments

**Comment by jduck** [comment]
Addressed comments.

**Pull request merged**